cmd/openshift-tests/disruption.go

```diff
@@ -0,0 +1,175 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	"github.com/openshift/origin/pkg/disruption/backend"
+	"github.com/openshift/origin/pkg/monitor"
+	"github.com/openshift/origin/pkg/monitor/apiserveravailability"
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/test/extended/util/disruption/controlplane"
+	"github.com/spf13/cobra"
+)
+
+// RunAPIDisruptionMonitorOptions sets options for api server disruption monitor
+type RunAPIDisruptionMonitorOptions struct {
+	Out, ErrOut io.Writer
+
+	ArtifactDir      string
+	LoadBalancerType string
+	ExtraMessage     string
+}
+
+func NewRunInClusterDisruptionMonitorOptions(ioStreams genericclioptions.IOStreams) *RunAPIDisruptionMonitorOptions {
+	return &RunAPIDisruptionMonitorOptions{
+		Out:    ioStreams.Out,
+		ErrOut: ioStreams.ErrOut,
+	}
+}
+
+func NewRunInClusterDisruptionMonitorCommand(ioStreams genericclioptions.IOStreams) *cobra.Command {
+	disruptionOpt := NewRunInClusterDisruptionMonitorOptions(ioStreams)
+	cmd := &cobra.Command{
+		Use:   "run-disruption",
+		Short: "Run API server disruption monitor",
+		Long: templates.LongDesc(`
+		Run a monitor which pings API servers and writes a file with intervals
+
```

Comment:
It doesn't look like the empty line here and below are needed for readability. Maybe remove them?

Reply:
Fixed